app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -203,18 +219,37 @@ def kill_middleware_domain_server(ems_ref)
       run_generic_operation(:Kill, ems_ref)
     end
 
-    def shutdown_middleware_server(ems_ref, params)
+    # server group ops
+    def start_middleware_server_group(ems_ref)
+      # blocking: boolean
+      run_generic_operation('Start Servers', ems_ref)
+    end
+
+    def stop_middleware_server_group(ems_ref, params)
+      # blocking: boolean
+      # timeout: int
```

Comment:
Oh duh I didn't see this `timeout = params[:timeout] || 0` below
I think I'd leave the comments ount and if timeout is optional, make `params = {}` so it is optional as well